k8s-deploy/Makefile

```diff
@@ -8,15 +8,25 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+
+ifeq ($(OS),Windows_NT)
+Buildmode = -buildmode=exe
```

Comment:
Please unify the variable name in makefile, with uppercase and underline combination.
For example `Buildmode` `BUILD_MODE`